### ISSUE TITLE
Remove // in the example?

### ISSUE THREAD

**Issue by jluttine** [rblaze/haskell-dbus#46]
See the example here: https://hackage.haskell.org/package/dbus

I couldn't find the definition of `(//)` anywhere and it's not in scope in my ghci if I import those three packages. Build also fails. But if I remove those three lines, then it builds. Maybe they should be removed?

**Comment by jluttine** [comment]
For completeness, here's a copy of that example:

```
{-# LANGUAGE OverloadedStrings #-}

import Data.List (sort)
import DBus
import DBus.Client

main = do
    client <- connectSession
    //
    -- Request a list of connected clients from the bus
    reply <- call_ client (methodCall "/org/freedesktop/DBus" "org.freedesktop.DBus" "ListNames")
        { methodCallDestination = Just "org.freedesktop.DBus"
        }
    //
    -- org.freedesktop.DBus.ListNames() returns a single value, which is
    -- a list of names (here represented as [String])
    let Just names = fromVariant (methodReturnBody reply !! 0)
    //
    -- Print each name on a line, sorted so reserved names are below
    -- temporary names.
    mapM_ putStrLn (sort names)
```

Those three lines with just `//` should be removed as far as I understand.

**Comment by rblaze** [comment]
Fixed in d5c3c8f5, will be published in the next release.

**Comment by jluttine** [comment]
Great, thanks a lot for the super-fast fix!
